src/App.tsx

```diff
@@ -1,24 +1,237 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable */
+
+import React, { useState, useEffect, ChangeEvent } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos, createTodos, deleteTodo, updateTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Error } from './types/Error';
+import { FilterValue } from './types/FiterValue';
 
-const USER_ID = 0;
+const USER_ID = 11260;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState(FilterValue.All);
+  const [errorMessage, setErrorMessage] = useState<Error>(Error.NoError);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [showNotification, setShowNotification] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const loadTodos = async () => {
+    try {
+      setIsLoading(true);
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch (error) {
+      setErrorMessage(Error.Loading);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const getFilteringTodos = () => {
```

Comment:
```suggestion
  const filteredTodos = useMemo(() => {
```
wrap function in useMemo. it wiil return filteredTodos instead call function in each places where you need actual todos.